src/node/node.ts

```diff
@@ -0,0 +1,21 @@
+import { CategoryItem } from 'src/category/item'
```

Comment:
put it back in the index file as it was before. we'll need to add all specific node implementations

Reply:
ignore this comment. it should be in the way you did